src/scenes/Partnerships/PartnershipForm/PartnershipForm.tsx

```diff
@@ -38,97 +36,60 @@ export const PartnershipForm = <
 >({
   partnership,
   ...rest
-}: PartnershipFormProps<T>) => {
-  const radioOptions = PartnershipAgreementStatusList.map((status) => (
-    <RadioOption
-      key={status}
-      label={displayPartnershipStatus(status)}
-      value={status}
-    />
-  ));
-
-  const typesCheckboxes = PartnershipTypeList.map((type: PartnershipType) => (
-    <div style={{ width: '50%' }} key={type}>
-      <CheckboxOption key={type} label={type} value={type} />
-    </div>
-  ));
-
-  return (
-    <DialogForm<T> {...rest} fieldsPrefix="partnership">
-      <SubmitError />
-      {!partnership && <OrganizationField name="organizationId" required />}
-      {partnership ? (
+}: PartnershipFormProps<T>) => (
+  <DialogForm<T> {...rest} fieldsPrefix="partnership">
+    {({ values }) => (
+      <>
+        <SubmitError />
+        {!partnership && <OrganizationField name="organizationId" required />}
         <SecuredField obj={partnership} name="types">
           {(props) => (
-            <CheckboxesField label="Types" row {...props}>
-              {typesCheckboxes}
-            </CheckboxesField>
+            <EnumField
```

Comment:
When I visit `/projects/:projectId/partnerships` for a project with partners, I get a crash and this error in the browser console:

```
PartnershipForm.tsx:47 Uncaught TypeError: Cannot read property 'types' of undefined
    at PartnershipForm.tsx:47
    at DialogForm.tsx:167
    at renderComponent (react-final-form.es.js:32)
    at ReactFinalForm (react-final-form.es.js:281)
```

Reply:
Herp derp. Thanks for catching that!